internal/vehicle/ford.go

```diff
@@ -63,11 +64,15 @@ func NewFordFromConfig(other map[string]interface{}) (api.Vehicle, error) {
 		vin:      strings.ToUpper(cc.VIN),
 	}
 
+	token, err := v.login()
+	if err == nil {
+		v.tokenSource = oauth.RefreshTokenSource((*oauth2.Token)(&token), v)
```

Comment:
NICE :)

Reply:
konnte leider nicht 1:1 die Lösung von Nissan/VW/ID übernehmen, da Ford dachte es wäre originell das Token nicht im "Authorization"-Header zu erwarten, sondern in "Auth-Token"